schemas/jsonschema/CircleAnnotation.json

```diff
@@ -1,7 +1,7 @@
 {
   "$comment": "Generated from CircleAnnotation by @foxglove/message-schemas",
   "title": "CircleAnnotation",
-  "description": "A circle annotation",
+  "description": "A circle annotation to be superimposed onto 2D images",
```

Comment:
Something about `to be superimposed` feels "off". Either it's the passive voice or maybe something else.

`A circle annotation for a 2D image`?

Reply:
That revision sounds good ^